Code/Mantid/instrument/Facilities.xml

```diff
@@ -606,6 +606,23 @@
   </instrument>
 </facility>
 
+<!--  Facility MLZ (FRMII) (Germany) -->
+<facility name="MLZ" FileExtensions=".nxs,.d_dat">
+  <instrument name="DNS">
+    <technique>Neutron Spectroscopy</technique> 
+    <technique>Reactor Direct Geometry Spectroscopy</technique>
+    <technique>TOF Direct Geometry Spectroscopy</technique>
+    <technique>Neutron Diffraction</technique>
+    <technique>Single Crystal Diffraction</technique>
+  </instrument>
+
+  <instrument name="TOFTOF">
+    <technique>Neutron Spectroscopy</technique>
+    <technique>Reactor Direct Geometry Spectroscopy</technique>
+    <technique>Time-of-flight spectroscopy</technique>
```

Comment:
The case of the last word should be fixed so it reads 'Time-of-flight Spectroscopy'. This will make it easier to find TOFTOF with the technique globbing facilities in Kernel.